pkgs/top-level/all-packages.nix

```diff
@@ -20805,7 +20805,45 @@ in
 
   openmw-tes3mp = libsForQt5.callPackage ../games/openmw/tes3mp.nix { };
 
-  openra = callPackage ../games/openra { lua = lua5_1; };
+  inherit (
```

Comment:
This is too much code for in `all-packages.nix`.

Reply:
I have moved that code into its own file, so now it is just a one liner in `all-packages.nix`.